src/Components/blazor/src/Microsoft.AspNetCore.Blazor.Templates/content/Components-CSharp/.template.config.src/template.json

```diff
@@ -1,113 +0,0 @@
-{
```

Comment:
The `Components-CSharp` template is redundant now, since the Razor Components template is now in the `.Templating` project.

Now was a good time to remove it from here, since it saved me the trouble of updating it to match the new dependency names.